public/components/event_analytics/explorer/events_views/docViewRow.tsx

```diff
@@ -36,7 +36,7 @@ interface IDocViewRowProps {
   onFlyoutOpen: (docId: string) => void;
 }
 
-export const DocViewRow = forwardRef((props: IDocViewRowProps, ref) => {
+export const FlyoutButton = forwardRef((props: FlyoutButtonProps, ref) => {
```

Comment:
rename the file as well?

Reply:
I would, but more in depth flyout changes are slated for p1 so I was going to get to that (along with many other changes) later.